setup.py

```diff
@@ -9,7 +9,7 @@
 # If adding requirements make sure to also add to requirements-dev.txt
 install_requires = [
     'botocore>=1.7.28, <=1.7.40',
-    'aiohttp>=2.0.4, <=2.3.0',
+    'aiohttp>=2.3.0, <=2.3.2',
```

Comment:
Push `<3.0.0` here.
All aiohttp `2.3` bugfixes should be backward compatible.
Honestly I don't expect many of them, maybe one or two with very minor changes.

Reply:
done